app.js

```diff
@@ -0,0 +1,205 @@
+    image: 'images/space.png',
+    title: 'Space travellers',
+    info: ['pair', 'APIs', '2022'],
+    description:
+      'Basic app for doing some reservations.',
+    skills: ['react', 'redux', 'bootstrap'],
+  },
+];
+
+const work = document.querySelector('#portfolio');
+function cardsBuilder(card, index) {
+  const {
+    image, title, info, description, skills,
+  } = card;
+  const infoHtml = info.map(
+    (i) => `<li class="canopy">${i}</li>
+    ${
+  i === info[info.length - 1]
+    ? ''
+    : '<li><img src="images/Counter.png" alt="counter" /></li>'
+}`,
+  ).join('');
+  const skillsHtml = skills.map(
+    (s) => `<li class="highlight font-1">${s}</li>`,
+  ).join('');
+  work.innerHTML += `
+  <div class="card card-${(index === 0 || index % 2 === 0) ? '1' : '2'}">
+        <a href="#"
+          ><img src="${image}" alt="Snapshoot-Portfolio" class="snapshot"
+        /></a>
+        <div class="${(index === 0 || index % 2 === 0) ? 'right' : 'left'}-cont">
+          <h3 class="tonic flex0">${title}</h3>
+          <ul class="ul-box1 flex0">
+            ${infoHtml}
+          </ul>
+          <p class="description-2 flex0">
+            ${description}
+          </p>
+          <ul class="ul-box2 flex0">
+            ${skillsHtml}
+          </ul>
+          <button class="highlight-2 font-4 flex0 btns">
+            See Project
+          </button>
+        </div>
+      </div>
+  `;
+}
+
+cards.forEach((card, index) => {
+  cardsBuilder(card, index);
+});
+
+const popupCards = [
+  {
+    title: 'Wishyacht',
+    info: ['frontend', 'backend', 'fullstack'],
+    description:
+      'This is a Web application where users should sign in to rent a Yacht from a list then reserve it for a determined period, users can then see their reservations or cancel them.',
+    skills: ['react', 'redux', 'rails'],
+    image: 'images/budget.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Budget App',
+    info: ['facebook', 'full stack dev', '2015'],
+    description:
+      'The Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.',
+    skills: ['ruby', 'rails', 'bootstrap'],
+    image: 'images/wishyacht.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Companies 3D',
+    info: ['frontend', 'Api', '2022'],
+    description:
+      "this is an informative/numeric web application that shows information about the canadian technical companies and their statistcs like the market cap and the trading status it also has search feature and for each company page it's unique root and responsive website",
+    skills: ['react', 'redux', 'bootstrap'],
+    image: 'images/company.png',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Space hub',
+    info: ['pair', 'APIs', '2022'],
+    description:
```

Comment:
- [ ] in this phase you should generate the projects section dynamically so that in the projects there is information but you go the pop-up window it shows different information so it would be better if this data here should be shown in the projects and also when I click on some project I should see the popup window with more details about the project 
- Project 
<img width="1194" alt="Screen Shot 2022-10-20 at 17 16 39" src="https://user-images.githubusercontent.com/61790587/197003295-63e3b1ee-3156-40cf-a606-203c27a78b19.png">
- Window pop-up
<img width="939" alt="Screen Shot 2022-10-20 at 17 16 45" src="https://user-images.githubusercontent.com/61790587/197003347-475ebc4c-78bc-4d9e-8294-f6e1e7c333fc.png">